modules/installation-creating-aws-vpc-subnets-edge.adoc

```diff
@@ -0,0 +1,86 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration//aws-compute-edge-zone-tasks..adoc (AWS edge zone tasks) 
+
+:_mod-docs-content-type: PROCEDURE
+[id="installation-creating-aws-vpc-subnets-edge_{context}"]
+= Creating subnets in AWS Local Zones or Wavelength Zones
+
+Before you configure a machine set for edge compute nodes in your {product-title} cluster, you must create the subnets in {zone-type}. Complete the following procedure for each edge zone that you want to deploy compute nodes to.
+
+You can use the provided CloudFormation template and create a CloudFormation stack. You can then use this stack to custom provision a subnet.
+
+[NOTE]
+====
+If you do not use the provided CloudFormation template to create your AWS infrastructure, you must review the provided information and manually create the infrastructure. If your cluster does not initialize correctly, you might have to contact Red Hat support with your installation logs.
+====
+
+.Prerequisites
+
+* You configured an AWS account.
+* You added your AWS keys and region to your local AWS profile by running `aws configure`.
+* You opted in to the {zone-type} group.
+
+.Procedure
+
+. Go to the section of the documentation named "CloudFormation template for the VPC subnet", and copy the syntax from the template. Save the copied template syntax as a YAML file on your local system. This template describes the VPC that your cluster requires.
+
+. Run the following command to deploy the CloudFormation template that in turn creates a stack of AWS resources that represent the VPC. You must enter the command on a single line. For a Local Zones environment, do not specify the public route table parameter in the command.
```

Comment:
- 1) Any reason for this comment/requirement?
- 2) Why not specify the route table ID? If not specified the stack will fail.

```suggestion
. Run the following command to deploy the CloudFormation template that in turn creates a stack of AWS resources that represent the VPC.
```

Reply:
Hi @mtulio . I have a command suggestion for checking the VPC ID value. Should I also add a step for the route table ID? Stating the importance of one parameter value over another, means other parameter seem less important.

Reply:
@dfitzmau :

> "I have a command suggestion for checking the VPC ID value"

I am trying to understand the benefit of this command here. As [mentioned](https://github.com/openshift/openshift-docs/pull/69537#discussion_r1488031256) in previously comments, if you are providing more verbose steps of getting VPC_ID, which is already described where to get this in the legend of the command, it may open why not providing for ALL variables (private rtb, public rtb, etc). So for minimalism, and prevent redundant information I suggest to not add the extra command.

> Should I also add a step for the route table ID?

IMO No

> Stating the importance of one parameter value over another, means other parameter seem less important.

Correct, for that reason I mentioned about the minimalism and improving the legend of the command, instead of adding extra steps.